Samples/MultiWindowSampleApplication/ViewModels/AuthorizedViewModel.cs

```diff
@@ -9,9 +9,9 @@ public class AuthorizedViewModel : NavigationViewModelBase
     {
         protected override void Navigate(NavigationViewModelBase navigationObject)
         {
-            if (navigationObject is AuthorizedViewModel authVM)
+            if (navigationObject is AuthorizedViewModel)
             {
-                if (VerifyNavigation(authVM))
+                if (VerifyNavigation((AuthorizedViewModel)navigationObject))
```

Comment:
Any particular reason you chose this way? The original seems like better readability.

Reply:
I'm using visual studio 2015, so I needed to downgrade the code from C# 7.0 to 6.0 to compile it.

Reply:
Okay. No problem. In case you haven't noticed I already accepted the PR and made the small changes. Thanks for the help on the AsyncCommand!